.dev.env

```diff
@@ -4,7 +4,8 @@ DJANGO_HIREDIS_CACHE_LOCATION=redis:6379
 DJANGO_ALLOWED_HOST=""
 DATABASE_URL=mysql://root:root@mysql/mc_dev
 EMAIL_URL=console://
-JWT_SECRET_KEY=ffffffff-ffff-ffff-ffff-ffffffffffffjwt
+JWT_AUTH_COOKIE=jwt
```

Comment:
can we make this the same name as what we use in accelerate? (JWT_COOKIE_NAME)

Reply:
I was debating whether this should be the same as accelerate or the same as drf-jwt's name. We can pick that up later.